configuration/pih/htmlforms/hiv/zl/section-hiv-history.xml

```diff
@@ -780,7 +779,25 @@
                 </label>
                 <obs conceptId="PIH:12999" rows="2" />
             </div>
+        </div>
+
+        <div class="section-container">
+            <h3><uimessage code="pihcore.keyPopulation"/></h3>
+            <repeat>
+                <template>
 
+                    <obs conceptId="PIH:11560"
+                         answerConceptId="{cId}"
+                         answerCode="pihcore.{mode}"
+                         style="checkbox"/>
+
+                </template>
+                <render mode="homosexual" cId="CIEL:160578"/>
```

Comment:
@jmbabazi Change the code here to match the messages.properties, and you'll be set.